numpy/random/generator.pyx

```diff
@@ -3421,18 +3434,35 @@ cdef class Generator:
         --------
         >>> mean = (1, 2)
         >>> cov = [[1, 0], [0, 1]]
-        >>> x = np.random.default_rng().multivariate_normal(mean, cov, (3, 3))
+        >>> rng = np.random.default_rng()
+        >>> x = rng.multivariate_normal(mean, cov, (3, 3))
         >>> x.shape
         (3, 3, 2)
 
+        We can use a different method other than the default to factorize cov:
+        >>> y = rng.multivariate_normal(mean, cov, (3, 3), method='cholesky')
+        >>> y.shape
+        (3, 3, 2)
+
+        We can also use a precomputed factor matrix of cov to speed up the
+        computation. This is very useful when one needs to generate random
+        values using the sample covariance matrix but different means.
+        >>> s, u = np.linalg.eigh(cov)
+        >>> A = u * np.sqrt(s)  # factor matrix of cov using Eigendecomposition.
+        >>> z = rng.multivariate_normal(mean, A, (3, 3), method='eigh', use_factor=True)
```

Comment:
Why does it matter what `method` is when `use_factor=True`?  Docstring should say that method is ignored when use_factor is True.

Reply:
It is not ignored. The factor matrices you get from the Cholesky method (especially if the upper triangular factor matrix is computed) require different treatment that those calculated from